### PR TITLE
Allow users to optionally specify API version

### PR DESCRIPTION
**Pull request by conr**
Curious to get some thoughts and opinions on this approach so far 🙂

Allow users to configure a specific version of the API.
By default, fallback to the latest version of the API hardcoded in `api_configuration.rb`.

```ruby
WhatsappSdk.configure do |config|
  config.access_token = ACCESS_TOKEN
  config.api_version = "v16.0"
end
```

### CODE REVIEW

**Review by ignacio-chiazzo — COMMENTED** [review]
This makes sense to me.

Consumers might also want to use old apis for some fields in case of a deprecation, but that can be added later. For example allow overriding an specific endpoint until the app migrated.

Could you add validations to the value passed? We could have a file that lists all the possible apis.

**Comment by conr** [comment]
> Could you add validations to the value passed? We could have a file that lists all the possible apis.

Yes, sounds good 👍🏻

File: config/api_versions.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+- v19.0
```
Comment (conr) [inline config/api_versions.yml:1]: Grabbed this list from Meta's [Graph API docs](https://developers.facebook.com/docs/graph-api/changelog/versions)

File: README.md [hunk 2]
```diff
@@ -41,6 +41,7 @@ To use `Messages`, `Media` or `PhoneNumbers`, you need to initialize the `Client
 # config/initializers/whatsapp_sdk.rb
 WhatsappSdk.configure do |config|
   config.access_token = ACCESS_TOKEN
+  config.api_version = API_VERSION # Optionally specify the API version to use (defaults to the latest version of the API).
```
Comment (ignacio-chiazzo) [inline README.md:44]: nit: could you specify the available versions with a link? Thanks

**Review by ignacio-chiazzo — COMMENTED** [review]
The code is looking good. I left one small suggestion. Thanks!

**Review by ignacio-chiazzo — APPROVED** [review]
Thanks! Great work.

**Pull request merged**